go/common/logging/logging.go

```diff
@@ -323,19 +335,38 @@ func (b *logBackend) getLogger(module string, extraUnwind int) *Logger {
 		logger = &log.SwapLogger{}
 	}
 
-	var keyvals []interface{}
-	if module != "" {
-		keyvals = append(keyvals, []interface{}{
-			"module",
-			module,
-		}...)
+	prefixes := []interface{}{
+		"ts", log.DefaultTimestampUTC,
+		"caller", log.Caller(defaultUnwind + extraUnwind),
+		"module", module,
```

Comment:
Slight simplification: We only used to set this for `module!=""`. Grepping shows this is currently always the case. I don't think that usage of `module==""` is encouraged, so if it ever does happen, it should be fine to explicitly draw attention to it with an empty string in the logs.